sda-commons-server-auth-testing/src/test/java/org/sdase/commons/server/opa/testing/test/AuthAndOpaBundleTestApp.java

```diff
@@ -43,14 +42,19 @@ public void run(AuthAndOpaBundeTestAppConfiguration configuration, Environment e
 
     @GET
     @Path("resources")
-    public Response get() throws IOException {
+    public Response get() {
       OpaJwtPrincipal principal = (OpaJwtPrincipal) securityContext.getUserPrincipal();
 
       PrincipalInfo result =
           new PrincipalInfo()
               .setName(principal.getName())
               .setJwt(principal.getJwt())
-              .setConstraints(principal.getConstraintsAsEntity(ConstraintModel.class));
+              .setSub(
+                  principal.getClaims() == null
+                      ? null
+                      : principal.getClaims().get("sub").asString())
```

Comment:
Also check the claim value for `null`?

Reply:
In a test where `sub` has a default value from the `AuthRule`? I think that is too much boilerplate.

Reply:
Alright. Did not see that this is just a test! :)